app/models/configuration_record.rb

```diff
@@ -0,0 +1,6 @@
+class ConfigurationRecord < ApplicationModel
```

Comment:
I don't believe this is necessary anymore.

Reply:
true